src/instrumentation_measurement/baseline.patch

```diff
@@ -0,0 +1,18 @@
+--- baseline.cpp	2022-03-28 14:59:23.985945207 +0200
```

Comment:
don't commit patches. Best if you open a branch at `imapp-pl/evmone` and push the changes there and just reference the branch here.